cli/outdated/outdated.go

```diff
@@ -0,0 +1,98 @@
+// This file is part of arduino-cli.
+//
+// Copyright 2020 ARDUINO SA (http://www.arduino.cc/)
+//
+// This software is released under the GNU General Public License version 3,
+// which covers the main part of arduino-cli.
+// The terms of this license can be found at:
+// https://www.gnu.org/licenses/gpl-3.0.en.html
+//
+// You can be released from the requirements of the above licenses by purchasing
+// a commercial license. Buying such a license is mandatory if you want to
+// modify or otherwise use the software for commercial activities involving the
+// Arduino software without disclosing the source code of your own applications.
+// To purchase a commercial license, send an email to license@arduino.cc.
+
+package outdated
+
+import (
+	"context"
+	"os"
+
+	"github.com/arduino/arduino-cli/cli/errorcodes"
+	"github.com/arduino/arduino-cli/cli/feedback"
+	"github.com/arduino/arduino-cli/cli/instance"
+	"github.com/arduino/arduino-cli/commands/core"
+	"github.com/arduino/arduino-cli/commands/lib"
+	rpc "github.com/arduino/arduino-cli/rpc/commands"
+	"github.com/arduino/arduino-cli/table"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// NewCommand creates a new `outdated` command
+func NewCommand() *cobra.Command {
+	outdatedCommand := &cobra.Command{
+		Use:   "outdated",
+		Short: "Lists cores and libraries that can be upgraded\n",
```

Comment:
```suggestion
		Short: "Lists cores and libraries that can be upgraded",
```

This created an empty row in the autocompletion like:

```
╰─$ ./arduino-cli
board            -- Arduino board commands.
burn-bootloader  -- Upload the bootloader.
cache            -- Arduino cache commands.
compile          -- Compiles Arduino sketches.
completion       -- Generates completion scripts
config           -- Arduino configuration commands.
core             -- Arduino core operations.
daemon           -- Run as a daemon on port 50051
debug            -- Debug Arduino sketches.
help             -- Help about any command
lib              -- Arduino commands about libraries.
outdated         -- Lists cores and libraries that can be upgraded
 
sketch           -- Arduino CLI sketch commands.
update           -- Updates the index of cores and libraries
upgrade          -- Upgrades installed cores and libraries.
upload           -- Upload Arduino sketches.
version          -- Shows version number of Arduino CLI.
```